earth_enterprise/src/installer/install_server.sh

```diff
@@ -192,22 +600,20 @@ modify_files()
   # if there are non-OSS installs in the system, it might help cleanup those.
 
   # a) Search and replace the below variables in /etc/init.d/geserver.
-  sed -i 's/IA_GEAPACHE_USER/GEAPACHEUSER_NAME/' $BININSTALLROOTDIR/geserver
-  sed -i 's/IA_GEPGUSER/GEPGUSER_NAME/' $BININSTALLROOTDIR/geserver
+  sed -i "s/IA_GEAPACHE_USER/$GEAPACHEUSER_NAME/" $BININSTALLROOTDIR/geserver
+  sed -i "s/IA_GEPGUSER/$GEPGUSER_NAME/" $BININSTALLROOTDIR/geserver
 
   # b) Search and replace the file ‘/opt/google/gehttpd/conf/gehttpd.conf’
-  sed -i 's/IA_GEAPACHE_USER/GEAPACHEUSER_NAME/' /opt/google/gehttpd/conf/gehttpd.conf
-  sed -i 's/IA_GEPGUSER/GEPGUSER_NAME/' /opt/google/gehttpd/conf/gehttpd.conf
+  sed -i "s/IA_GEAPACHE_USER/$GEAPACHEUSER_NAME/" /opt/google/gehttpd/conf/gehttpd.conf
+  sed -i "s/IA_GEPGUSER/$GEPGUSER_NAME/" /opt/google/gehttpd/conf/gehttpd.conf
+  sed -i "s/IA_GEGROUP/$GRPNAME/" /opt/google/gehttpd/conf/gehttpd.conf
 
```

Comment:
$GEAPACHEUSER_NAME, $GEPGUSER_NAME and $GEPGUSER_NAME should replace the GEFusionUser, GEFusionGroup specified in parse_arguments.  The show_help method should also be updated to reflect this.